code/script.js

```diff
@@ -1,53 +1,230 @@
+      id="name-input"
+      type="text"
+    />
+    <button id="send-btn" class="send-btn" type="submit">→</button>
+  </form>`
+  const sendBtn = document.getElementById("send-btn")
+  if (sendBtn) {
+    sendBtn.onclick = handleNameInput
+  }
+}
+// Moved the message function here 👇🏻
 
-// A function that will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  // The if statement checks if the sender is the user and if that's the case it inserts
-  // an HTML section inside the chat with the posted message from the user
-  if (sender === 'user') {
+  if (sender === "user") {
+    console.log("message is:", message)
+    console.log("sender is:", sender)
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/child-2707415_1280.png" alt="User" />  
       </section>
     `
-    // The else if statement checks if the sender is the bot and if that's the case it inserts
-    // an HTML section inside the chat with the posted message from the bot
-  } else if (sender === 'bot') {
+    // added the audio that you linked for bot 👇🏻
+    audioUser.play()
+  } else if (sender === "bot") {
+    console.log("message is:", message)
+    console.log("sender is:", sender)
+
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/jaws-4659701_1280.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
+    // added the audio that you linked for bot 👇🏻
+    audio.play()
   }
 
   // This little thing makes the chat scroll to the last message when there are too many to
   // be shown in the chat box
+
   chat.scrollTop = chat.scrollHeight
 }
+// This is the function to handle the name submission👇🏻
 
-// A function to start the conversation
-const greetUser = () => {
-  // Here we call the function showMessage, that we declared earlier with the argument:
-  // "Hello there, what's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, what's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆 and see what happens
+const handleNameInput = (event) => {
+  event.preventDefault()
+
+  // Store the value in a variable so we can access it after we
+  // clear it from the input
+
+  const nameInput = document.getElementById("name-input")
+  const name = nameInput.value
+  if (/^[a-zA-z]/.test(name)) {
+    showMessage(`I'm ${name}!`, "user")
+    nameInput.value = ""
+    setTimeout(() => firstChoice(name), 500)
+  } else {
+    showMessage(`Please fill in a valid name with letters.`, "bot")
+  }
 }
 
-// Eventlisteners goes here 👇
+// second message from bot
+
+const firstChoice = (name) => {
```

Comment:
Maybe this function could get a more descriptive name, like askForHairCutLength?